js-exercises/sequential-promise/sequentialPromise.js

```diff
@@ -0,0 +1,14 @@
+const sequentialPromise = fns => {
+    const resultantPromise = fns.reduce((promise, fn) => {
+      return promise.then(result => {
+        return fn(result[result.length - 1]).then(res => {
+          return result.concat(res);
+        });
+      });
+    }, Promise.resolve([]));
+    return resultantPromise.then(res => {
+      return res[res.length - 1];
+    });
+  };
+  
+  export { sequentialPromise };
```

Comment:
Why isn't the code properly indented? Highly recommend using something like Prettier to auto format your code.